pages/common/unzip.md

```diff
@@ -13,3 +13,7 @@
 - List the contents of a zip file without extracting:
 
 `unzip -l {{file}}`
+
+- Extract the contents of a zip file to the console:
```

Comment:
The wording can be improved a bit. Contents of a zip file is the unzipped file. We need to clarify that it will display the contents of the unzipped file, not the contents of the zip file, in which case it becomes the same as `-l`.

How about - `Display the contents of the file(s) inside a zip file to the console`